src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog-test_task/blob/solution-to-review/checklist.md) #8
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.